<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>HoloLens MDL2 Assets</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>双色球系统</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>10</y>
     <width>161</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resource.qrc">:/img/qq.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>401</width>
     <height>151</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="password_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HoloLens MDL2 Assets</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>账号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="login_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HoloLens MDL2 Assets</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>密码：</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>540</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>15</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>版权所有：AdamLi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>440</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>671</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文隶书</family>
     <pointsize>100</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>双色球系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_test">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>630</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bookman Old Style</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>个人网站</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
